app/api/test-email/route.ts

```
@@ -45,13 +64,13 @@ export async function POST(request: Request) {
     return NextResponse.json({ 
       success: true, 
       messageId: testEvent.Records[0].ses.mail.messageId,
-      previewBranchName,
+      currentBranch,
       lambdaResponse: response
     });
   } catch (error) {
-    console.error("Error testing Lambda:", error);
+    console.error("Error testing email processing:", error);
```

We should return this error into the frontend